packages/config/src/base.config.ts

```diff
@@ -58,6 +58,23 @@ export class ConfigInstance {
         return output;
     }
 
+    /**
+     * Prefix a dynamoDb id with the provided prefix if it doesnt already start with it.
+     */
+    prefix(prefix: ConfigPrefix, id: string): string {
+        if (id === '') return id;
```

Comment:
Should we return id here? I think the empty id is invalid.

Reply:
This was just copy and pasted from the other spot, I haven't changed the logic